docs/source/markdown/podman-info.1.md

```diff
@@ -30,128 +30,212 @@ Run podman info with plain text response:
 ```
 $ podman info
 host:
-  BuildahVersion: 1.4-dev
-  Conmon:
-    package: Unknown
-    path: /usr/libexec/podman/conmon
-    version: 'conmon version 1.12.0-dev, commit: d724f3d54ad2d95b6de741085d4990190ebfd7ff'
-  Distribution:
-    distribution: fedora
-    version: "28"
-  MemFree: 1271083008
-  MemTotal: 33074233344
-  OCIRuntime:
-    package: runc-1.0.0-51.dev.gitfdd8055.fc28.x86_64
-    path: /usr/bin/runc
-    version: 'runc version spec: 1.0.0'
-  SwapFree: 34309664768
-  SwapTotal: 34359734272
   arch: amd64
+  buildahVersion: 1.15.0
```

Comment:
is this version right?

Reply:
I removed the -dev flag, to make it look better, since we will probably not update this when we deliver podman 2.0.

Reply:
Ah, thanks.  I was a bit concerned we were already spitting out v1.15 somehow.